### PR TITLE
HTCONDOR-1637: Finish Ubuntu18 removal

### PR DESCRIPTION
**Pull request by timtheisen**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [x] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [x] Check for correctness of change
- [x] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [x] Check for documentation, if needed
- [x] Check for version history, if needed
- [x] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: nmi_tools/glue/build/build_uw_deb.sh [hunk 1]
```diff
@@ -83,13 +83,10 @@ if [ "$VERSION_CODENAME" = 'bullseye' ]; then
     true
 elif [ "$VERSION_CODENAME" = 'bookworm' ]; then
     dch --distribution $dist --nmu 'place holder entry'
-elif [ "$VERSION_CODENAME" = 'bionic' ]; then
-    true
 elif [ "$VERSION_CODENAME" = 'focal' ]; then
-    dch --distribution $dist --nmu 'place holder entry'
+    true
```
Comment (Todd-L-Miller) [inline nmi_tools/glue/build/build_uw_deb.sh:87]: This changes `focal` from `dch --distribution $dist --nmu 'place holder entry'` to `true`.  Can't tell if that's intentional or not.

**Comment by Todd-L-Miller** [comment]
Looks like documentation and release notes were done on another ticket.

**Comment by timtheisen** [comment]
Docs and release notes were done in another PR. I just noticed vestiges that were not purged at that time. Hench this PR.

File: nmi_tools/glue/build/build_uw_deb.sh (hunk 1 above)
Comment (Todd-L-Miller) [reply nmi_tools/glue/build/build_uw_deb.sh:87]: Explained in-person.  Thanks.

**Review by Todd-L-Miller — APPROVED** [review]

**Pull request merged**